### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mpr87**
D4 Implementation 1

### CODE REVIEW

File: Implementation.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+### 7. Readme File
+
+**You should also prepare your repository for receiving new contributors. See an example
+of a Readme.md file at [https://gist.github.com/PurpleBooth/109311bb0361f32d87a2](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
+Besides the Readme.md file, your repository should contain a CONTRIBUTING.md
+file, a LICENSE file, and a CODE_OF_CONDUCT.md file. Search online for some
+examples of these files. In this section of the deliverable, put links to these files on GitHub.**
+
+[https://github.com/ChrisKeefe/DontPanic/blob/master/README.md](https://github.com/ChrisKeefe/DontPanic/blob/master/README.md)
```
Comment (ChrisKeefe) [inline Implementation.md]: These are getting lumped together - can you separate them into separate lines? Maybe a bullet list?

File: Implementation.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+Because Firebase runs the services that would normally run on a container, a Docker image is not needed. This highlights another advantage of choosing Firebase for our application and has been approved by Dr. Gerosa.
+
+### 6. Licensing
+
+**Inform the license you adopted for your source code (remember to configure GitHub
+accordingly) and why you adopted this license. For more information, check [https://choosealicense.com/](https://choosealicense.com/) and [http://redhat.slides.com/glikins/open-sourcelicensing-101#/](http://redhat.slides.com/glikins/open-sourcelicensing-101#/)**
+
+**MIT License**
+
+[https://choosealicense.com/licenses/mit/](https://choosealicense.com/licenses/mit/)
```
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [MIT license overview](https://choosealicense.com/licenses/mit/)

```

File: Implementation.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+[https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE](https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE)
```
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [Our License](https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE)

```

File: Implementation.md (hunk 1 above)
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [README.md](https://github.com/ChrisKeefe/DontPanic/blob/master/README.md)

```

File: Implementation.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+### 7. Readme File
+
+**You should also prepare your repository for receiving new contributors. See an example
+of a Readme.md file at [https://gist.github.com/PurpleBooth/109311bb0361f32d87a2](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
+Besides the Readme.md file, your repository should contain a CONTRIBUTING.md
+file, a LICENSE file, and a CODE_OF_CONDUCT.md file. Search online for some
+examples of these files. In this section of the deliverable, put links to these files on GitHub.**
+
+[https://github.com/ChrisKeefe/DontPanic/blob/master/README.md](https://github.com/ChrisKeefe/DontPanic/blob/master/README.md)
+[https://github.com/ChrisKeefe/DontPanic/blob/master/CONTRIBUTING.md](https://github.com/ChrisKeefe/DontPanic/blob/master/CONTRIBUTING.md)
```
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [CONTRIBUTING.md](https://github.com/ChrisKeefe/DontPanic/blob/master/CONTRIBUTING.md)

```

File: Implementation.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+[https://github.com/ChrisKeefe/DontPanic/blob/master/CODE_OF_CONDUCT.md](https://github.com/ChrisKeefe/DontPanic/blob/master/CODE_OF_CONDUCT.md)
```
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [CODE_OF_CONDUCT.md](https://github.com/ChrisKeefe/DontPanic/blob/master/CODE_OF_CONDUCT.md)

```

File: Implementation.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+[https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE](https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE)
```
Comment (ChrisKeefe) [inline Implementation.md]:
```suggestion
- [LICENSE](https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE)

```

**Review by ChrisKeefe — CHANGES_REQUESTED** [review]
Solid, Mason. A few minor comments inline, then we're good to go!

**Review by mpr87 — COMMENTED** [review]
update md

**Review by ChrisKeefe — DISMISSED** [review]
LGTM

**Review by ChrisKeefe — APPROVED** [review]
LGTM!

**Pull request merged**